garrysmod/lua/includes/modules/widget.lua

```diff
@@ -145,6 +141,24 @@ hook.Add( "PostDrawEffects", "RenderWidgets", function()
 
 end )
 
+hook.Add( "OnEntityCreated", "CreateWidgets", function( ent )
+	timer.Simple( 1, function( ) -- why is this needed? no one may ever know!

```

Comment:
timer with 0 delay should only be called on the same frame if you are adding the timer from another timer.